Resources/Prototypes/Species/reptilian.yml

```
@@ -56,13 +56,13 @@
       points: 1
       required: false
     Chest:
-      points: 3
+      points: 4 # imp
```

```suggestion
      points: 4 # imp 3<4
```
Note original value for future maintenance work.